source/exercises/mailroom-part3.rst

```diff
@@ -0,0 +1,18 @@
+.. _exercise_mailroom_part3_exceptions:
```

Comment:
This one should probably be fleshed out a bit more -- did your students figure out what to do?